spec/requests/users_spec.rb

```diff
@@ -1,9 +1,55 @@
-# frozen_string_literal: true
-
 require "rails_helper"
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
I think it's better not to disable rubocop checks.

Reply:
What you think will be better in this case: to change cop config or to refactor code? :)

Reply:
@RasimKhusaenov It depends. If you can change your code, you can do it. If you think the rule is redundant, you can change the Rubocop config.